test/e2e/node/runtimeclass.go

```diff
@@ -115,6 +121,47 @@ var _ = ginkgo.Describe("[sig-node] RuntimeClass", func() {
 		framework.ExpectEqual(nodeSelector, pod.Spec.NodeSelector)
 		gomega.Expect(pod.Spec.Tolerations).To(gomega.ContainElement(tolerations[0]))
 	})
+
+	ginkgo.It("should run a Pod requesting a RuntimeClass with scheduling without taints [NodeFeature:RuntimeHandler] ", func() {
```

Comment:
Does this actually need the `[NodeFeature:RuntimeHandler]`? If it doesn't have the runtime handler properly configured, the pod will fail, but the test doesn't actually check that the pod is running, so that might be ok?

Reply:
hm.. this is mostly a copy of a test above that has the same logic with the addition of taints. so I didn't put too much thoughts to it.

Looking at code we can replace `WaitForPodNotPending` with `WaitTimeoutForPodReadyInNamespace` to ensure that pod will actually run. But maybe a better solution would be to change `[NodeFeature:RuntimeHandler]` to `[NodeFeature:RuntimeClass]` as the test actually validates this part of the logic. Since RuntimeClass is about to GA, no need to prefix it with the test tag so I will just remove the mention of any feature.